source/chapters/configuration.rst

```diff
@@ -50,12 +53,39 @@ inputs and outputs to be used.
   input. The sample rate value should be set to the sample rate of your audio
   interface. By default, Mixxx tries the system default first, which is most
   likely 44.1 kHz. Otherwise, Mixxx will pick a different default.
-
+        
 * **Audio buffer**: Also known as latency, this is the lag time in milliseconds
   that it takes for Mixxx to process your input. Lower latency means Mixxx
   will be more responsive but on slower computers and cheaper sound cards it
   might cause glitches.
 
+* **Multi-Soundcard Synchronization**: Mixxx is able to use two or more soundcards 
+  at a time, each with its own clock. The Mixxx engine is driven by the Master 
+  soundcards. Here you can select the synchronization used for the other 
+  soundcards to avoid buffer overflows or underflows.
+
+* **Keylock/Pitch-Bending Engine**: Allows you to select the engine for all 
+  independent tempo and pitch changes like keylock. Use Soundtouch on old devices 
```

Comment:
It'd be good to mention that Rubberband sounds better:
"Rubberband sounds better than Soundtouch, however old computers and netbooks may not be able to handle using Rubberband without audible glitches."